tests/prod/prod_sanity_suite_test.go

```diff
@@ -326,3 +326,17 @@ func TestVirtualDevice2PortCreateConnection_DIGP(t *testing.T) {
 	})
 	terraform.Apply(t, terraformOptions)
 }
+
+func TestVirtualDevice2AWSCreateConnection_DIGP(t *testing.T) {
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../../tests/examples-without-external-providers/virtual-device-2-aws-connection",
```

Comment:
The prod tests should use the example in the main examples directory. The intention is for it to be a complete test including the CSP for prod tests unless there's an actual issue with that happening for some product reason.

Reply:
Use the main example for the test.

Reply:
We won't be able to use main example for the PROD Sanity Test because we have integrated that example with Network Edge resources and it will take 15-20 mins to create virtual device.

Reply:
Yes. And, I see all the other test cases are done the same way (pointing to examples in tests)

Reply:
All good 👍 

Please call that out in your PR.